### PR TITLE
Fix: Documentation typos and fixes

### LINKED ISSUE #306: Wrong command in 40-clusters.md
In doAzureParallel/docs/40-clusters.md it gives the command
`cluster <- listClusters())`
which should be
`getClusterList()`

### PR DESCRIPTION
**Pull request by brnleehng**
#306 
#303 
#304 
#305

### CODE REVIEW

File: docs/04-azure-requirements.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Azure Requirements
+
+To run your R code across a cluster in Azure, we'll need to get keys and account information.
+
+### Setup Azure Account
+First, set up your Azure Account ([Get started for free!](https://azure.microsoft.com/en-us/free/))
+
+Once you have an Azure account, you'll need to create the following two services in the Azure portal:
+  - Azure Batch Account ([Create an Azure Batch Account in the Portal](https://docs.microsoft.com/en-us/azure/Batch/batch-account-create-portal))
```
Comment (zfengms) [inline docs/04-azure-requirements.md]: Azure Batch Account line and Storage Account line don't have the same indentation

File: docs/04-azure-requirements.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- Azure Storage Account (this can be created with the Batch Account)
+
+### Get Keys and Account Information
+For your Azure Batch Account, we need to get:
+  - Batch Account Name
```
Comment (zfengms) [inline docs/04-azure-requirements.md]: Indentation is different than next 2 lines.

File: docs/04-azure-requirements.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- Batch Account URL
+- Batch Account Access Key
+
+This information can be found in the Azure Portal inside your Batch Account:
+
+  ![Azure Batch Acccount in the Portal](./vignettes/doAzureParallel-azurebatch-instructions.PNG "Azure Batch Acccount in the Portal")
+
+For your Azure Storage Account, we need to get:
+  - Storage Account Name
```
Comment (zfengms) [inline docs/04-azure-requirements.md]: Indentation issue

File: docs/04-azure-requirements.md (hunk 1 above)
Comment (brnleehng) [reply docs/04-azure-requirements.md]: Updated

File: docs/04-azure-requirements.md (hunk 2 above)
Comment (brnleehng) [reply docs/04-azure-requirements.md]: Updated

**Pull request merged**